src/steps/02-EvaluationCriteria/JandA/MinimumRequirements.vue

```diff
@@ -1,25 +1,188 @@
 <template>
-  <v-form ref="form" lazy-validation>
-    <v-container fluid class="container-max-width">
-      <v-row>
-        <v-col class="col-12">
-          <h1 class="mb-3">
-            Tell us about your minimum government requirements
-          </h1>
-          <p>
-            Intro paragraph
-          </p>
-        </v-col>
-      </v-row>
-    </v-container>    
-  </v-form>
+  <div>
+    <v-form ref="form" lazy-validation>
+      <v-container fluid class="container-max-width">
+        <v-row>
+          <v-col class="col-12">
+            <h1 class="mb-3">
+              Tell us about your minimum government requirements
+            </h1>
+            <p>
```

Comment:
add `class="mb-10` to `p` tag to match bottom margin in Figma. Also add class `copy-max-width` to keep its width in check.
![Screen Shot 2023-04-25 at 3 24 47 PM](https://user-images.githubusercontent.com/90333349/234382712-074e593a-9e22-414b-b8a0-8f7daecdc629.png)